docs/api.md

```diff
@@ -1310,25 +1310,31 @@ Generate a bash completion script. Users of your application can install this
 script in their `.bashrc`, and yargs will provide completion shortcuts for
 commands and options.
 
-.showHelp(consoleLevel='error')
+.showHelp([consoleLevel | printCallback])
 ---------------------------
 
-Print the usage data using the [`console`](https://nodejs.org/api/console.html) function `consoleLevel` for printing.
```

Comment:
This change sould not remove the default value of `consoleLevel` in the signature.

Reply:
It's not removing the default value - see the second paragraph under "if no argument is provided". I fixed it according to the [comment](https://github.com/yargs/yargs/pull/1386#discussion_r309314320) by @bcoe. LMK if you have another specific change in mind.

Reply:
You're right, sorry.